src/BlazorWebView/src/Maui/build/Microsoft.AspNetCore.Components.WebView.Maui.props

```diff
@@ -1,5 +1,15 @@
 ﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
   <PropertyGroup>
     <StaticWebAssetsEnabled Condition="'$(_OuterIntermediateOutputPath)' != ''">false</StaticWebAssetsEnabled>
+    <UsingMicrosoftAspNetCoreComponentsWebViewMaui>true</UsingMicrosoftAspNetCoreComponentsWebViewMaui>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- .NET MAUI Blazor Hybrid -->
+    <_MauiMinimumSupportedOSPlatformVersion Condition="'$(_MauiMinimumSupportedOSPlatformVersion)' == '' and $([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'">24.0</_MauiMinimumSupportedOSPlatformVersion>
+    <_MauiDefaultSupportedOSPlatformVersion Condition="'$(_MauiDefaultSupportedOSPlatformVersion)' == '' and $([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'">24.0</_MauiDefaultSupportedOSPlatformVersion>
+    <_MauiMinimumSupportedOSPlatformVersion Condition="'$(_MauiMinimumSupportedOSPlatformVersion)' == '' and $([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">14.0</_MauiMinimumSupportedOSPlatformVersion>
```

Comment:
These are the values from the Blazor templates from .NET 8, so we are keeping those the same.